### DISCUSSION TITLE
how can i uninstall this $hit?

### DISCUSSION THREAD

**Discussion by eriksenmarshal** [Tyrrrz/YoutubeDownloader#255] [Q&A]
after the installation, there is no where i can find the program rather the zip file! how can i uninstall this scam app?

**Comment by ScottsplaysSus** [comment]
Where did you download the zip file to your PC(most browsers make `C:\Users\(your name)\This PC\Downloads` the default path for downloads)? You can download 7-zip to extract the contents from a zip file to your desktop directory, and the app is inside the zip file. Don't download the source code zip file though, download the YoutubeDownloader.zip instead :) 

Edit: It's not a scam app, it's free and open-source, you can check the code yourself if you want. I use this app regularly and have no issues using it :)

I'm not an expert in this, but I hope I can help :D
